databrowser/src/domain/datasets/ui/tableView/toolBox/SearchAndFilterToolBoxPanel.vue

```diff
@@ -54,7 +54,18 @@ SPDX-License-Identifier: AGPL-3.0-or-later
           @filter="updateFilter(index, filter.operator, $event, true)"
         />
       </ToolBoxCardBody>
-      <ToolBoxCardBody>
+      <ToolBoxCardBody class="flex justify-between">
+        <ButtonCustom
+          class="flex items-center gap-2 p-2 py-1"
+          :size="Size.xs"
+          :variant="Variant.solid"
+          @confirmed-value="emit('filter', $event)"
```

Comment:
The `ButtonCustom` does not emit a `@confirmed-value` element, maybe that's some kind of leftover?